### DISCUSSION TITLE
Store threads in cloud storage

### DISCUSSION THREAD

**Discussion by Unlok-Simon** [janhq/jan#2851] [Feature Ideas]
It would be useful to be able to change the directory for the storage of threads and use a synchronised cloud storage (e.g. onedrive, google drive etc..). This would enable having threads go across devices. I was going to try doing this with the data folder location but realised it could be a lot of data given the model settings folder would also be there.

**Comment by davidjimenez75** [comment]
IMHO storing the threads in {threadname}.txt files on the User Documents/Jan/ folder will be great for the standard user, this feature must be enable by default on installation (but could be disabled for privacy purposes in settings). The user later can rename the files as they want and easily open for copy and paste with notepad.

**Reply by rlneumiller** [reply]
For privacy it might be best to only store the threads after prompting the user to make the choice one way or another (and perhaps have the ubiquitous "Remember my selection" checkbox).
